.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+# Reference: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # "npm" is the YAML value for Yarn
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    pull-request-branch-name:
+      # By default, Dependabot separates sections of its branch names with `/`, which fails our
+      # ‘Validate branch name’ CI step
+      #
+      # This makes it use
+      #   dependabot-npm_and_yarn-express-4.19.2
+      # instead of
+      #   dependabot/npm_and_yarn/express-4.19.2
```

Comment:
Not entirely sure what to do about `.` being in the branch names too 😕

The only alternative I can think of is not enforcing our usual rules when a branch name starts with `dependabot`; but this means devs have to remember to explicitly name feature branch deployments if we need to deploy a Dependabot branch to test